integration-tests/src/common/mod.rs

```diff
@@ -20,7 +19,7 @@ pub(crate) static BITCOIN_CONFIG: Lazy<Ini> =
 /// Bitcoin params client to local node.
 ///
 /// Parameters are read from local ./configs/bitcoind.conf file.
-pub(crate) static BITCOIN_CLIENT_PARAMS: Lazy<(String, bitcoincore_rpc::Auth)> = Lazy::new(|| {
+pub(crate) static BITCOIN_CLIENT_PARAMS: Lazy<(String, String, String)> = Lazy::new(|| {
```

Comment:
Maybe it is worth creating a structure with these three strings?

Reply:
This thing is internal for tests only, so I think we can keep it as it is

Reply:
Agreed